modules/nw-kuryr-cleanup.adoc

```diff
@@ -12,9 +12,15 @@ plugin, you must clean up the resources that Kuryr created previously.
 [NOTE]
 ====
 The clean up process relies on a Python virtual environment to ensure that the package versions that you use support tags for Octavia objects. You do not need a virtual environment if you are certain that your environment uses at minimum:
+
 * `openstacksdk` version 0.54.0
+
 * `python-openstackclient` version 5.5.0
-* `python-octaviaclient` version 2.3.0 
+
+* `python-octaviaclient` version 2.3.0
+
+If you decide to use those particular versions, be sure to pull `python-neutronclient` prior to version 9.0.0, as it will prevent you from accessing trunks.
```

Comment:
```suggestion
If you decide to use these versions, be sure to pull `python-neutronclient` prior to version 9.0.0, as it prevents you from accessing trunks.
```

Reply:
Avoid using future tense

Reply:
Can "those particular" be swapped for "these" 🤔 I think that makes it clearer that you're referring to the preceding list.

Reply:
Ack